README.md

```diff
@@ -1 +1,42 @@
-# Smart-Fones-Store
\ No newline at end of file
+# Smartphone Store:-
+## :globe_with_meridians: LINKS:-
+- **Figma link:**[here](https://www.figma.com/file/4KnQRIqQPOCFlBxLi7IfXx/Untitled?node-id=193%3A872)
+- **Live link:**
+- **API documentation:** 
+1- [here(github)](https://docs.github.com/en/rest/guides/getting-started-with-the-rest-api)
+2-[here(brands phone api)](api-mobilespecs.azharimm.site/v2/brands)
+3- [here (phone api)](https://api-mobilespecs.azharimm.site/v2/brands/apple-phones-48?page=2)
+##:space_invader: The problem:
+Because of the epidemic that the world witnessed (Covid 19), which led to the deterioration of the economy and the closure of millions of different shops, which called for the establishment of electronic stores.
```

Comment:
need format put an enter after API link 

<img width="1076" alt="Screen Shot 2022-03-24 at 12 13 49 PM" src="https://user-images.githubusercontent.com/47992412/159894425-80f88dc7-b2e3-4ccf-8796-381de72fedbc.png">

3- [here (phone api)](https://api-mobilespecs.azharimm.site/v2/brands/apple-phones-48?page=2)

## :space_invader: The problem:
Because of the epidemic that the world witnessed (Covid 19), which led to the deterioration of the economy and the closure of millions of different shops, which called for the establishment of electronic stores.